.github/workflows/go.yml

```diff
@@ -22,3 +22,18 @@ jobs:
         run: go build -v ./...
       - name: End-to-end
         run: make e2e
+  sign:
+    runs-on: macos-latest
+    needs: build
+    steps:
+      - name: Import Code-Signing Certificates
```

Comment:
use the action group austin

https://github.com/shopmonkeyus/hitch/blob/main/.github/workflows/build.yml#L19

Reply:
Does there need to be extra configuration to let this repo access that runner group? I mirrored the config to hitch in this commit: https://github.com/shopmonkeyus/eds-server/pull/77/commits/c28cecbe241c2335836c8c38a0485999e05208fb, but it looks like it's stuck looking for a runner from that runner group: https://github.com/shopmonkeyus/eds-server/actions/runs/7413636711/job/20173041862?pr=77